website/static/js/nodeControl.js

```diff
@@ -46,6 +46,18 @@ var ProjectViewModel = function(data) {
     self.user = data.user;
     self.nodeIsPublic = data.node.is_public;
     self.nodeType = data.node.node_type;
+    self.instLogoPath = ko.observable('');
+
+    if (data.node.institution) {
+        $osf.ajaxJSON(
```

Comment:
As mentioned in a previous comment, avoid HTTP work upon initialization of ViewModels. Instead, put it in a method. You could then invoke the method in `NodeControl.prototype.init`.